assets/js/hooks/cell.js

```diff
@@ -333,7 +333,7 @@ const Cell = {
 
       if (this.currentEditor()) {
         this.currentEditor().focus();
-        this.scrollEditorCursorIntoViewIfNeeded();
+        this.scrollEditorCursorIntoViewIfNeeded("instant");
```

Comment:
I think we always want to be instant. I don't think there is a moment where we want it to be smooth, right @hugobarauna?